Library/Homebrew/cmd/style.rb

```diff
@@ -8,18 +23,93 @@ def style
       ARGV.formulae.map(&:path)
     end
 
+    Homebrew.failed = check_style_and_print(target, :fix => ARGV.flag?("--fix"))
+  end
+
+  # Checks style for a list of files, printing simple Rubocop output.
+  # Returns true if violations were found, false otherwise.
+  def check_style_and_print(files, options = {})
+    check_style_impl(files, :print, options)
+  end
+
+  # Checks style for a list of files, returning results as a RubocopResults
+  # object parsed from its JSON output.
+  def check_style_json(files, options = {})
+    check_style_impl(files, :json, options)
+  end
+
+  def check_style_impl(files, output_type, options = {})
+    fix = options[:fix]
     Homebrew.install_gem_setup_path! "rubocop", "0.39"
 
-    args = [
-      "--format", "simple", "--force-exclusion", "--config",
-      "#{HOMEBREW_LIBRARY}/.rubocop.yml",
+    args = %W[
+      --force-exclusion
+      --config #{HOMEBREW_LIBRARY}/.rubocop.yml
     ]
+    args << "--auto-correct" if ARGV.homebrew_developer? && fix
+    args += files
 
-    args << "--auto-correct" if ARGV.homebrew_developer? && ARGV.flag?("--fix")
+    case output_type
+    when :print
+      system "rubocop", "--format", "simple", *args
```

Comment:
Can we add the RuboCop option `--display-cop-names` (short `-D`) here? It makes it a lot easier to look up information when researching how to resolve a style violation (or what needs to be adjusted, if changing the code is not desirable). Compare:

```
$ rubocop --format simple --force-exclusion --config Library/.rubocop.yml Library/brew.rb
== Library/brew.rb ==
W:139:  1: Avoid rescuing the Exception class. Perhaps you meant to rescue StandardError?
```

with

```
$ rubocop -D --format simple --force-exclusion --config Library/.rubocop.yml Library/brew.rb
== Library/brew.rb ==
W:139:  1: Lint/RescueException: Avoid rescuing the Exception class. Perhaps you meant to rescue StandardError?
```